x/yarpctest/request.go

```diff
@@ -39,3 +42,9 @@ func Procedure(procedure string) *types.Procedure {
 func ShardKey(key string) *types.ShardKey {
 	return &types.ShardKey{ShardKey: key}
 }
+
+// ShardKey specifies that "shard key" header for a request. It is a shared
+// option across different requests.
+func Chooser(f func(peer.Identifier, peer.Transport) (peer.Chooser, error)) *types.ChooserFactory {
```

Comment:
comment doesn't match – this is causing the build/lint failure:
```
/yarpc/x/yarpctest/request.go:46:1: comment on exported function Chooser should be of the form "Chooser ..."
```

```suggestion
// Chooser optionally specifies a peer.Chooser to be used with the outbound.
func Chooser(f func(peer.Identifier, peer.Transport) (peer.Chooser, error)) *types.ChooserFactory {
```